src/algebra/lie_algebra.lean

```diff
@@ -207,6 +207,12 @@ instance : has_one (L₁ →ₗ⁅R⁆ L₁) := ⟨{ map_lie := by simp, ..(1 :
 
 instance : inhabited (L₁ →ₗ⁅R⁆ L₂) := ⟨0⟩
 
+@[ext] lemma morphism.ext {f g : L₁ →ₗ⁅R⁆ L₂} (h : ∀ x, f x = g x) : f = g :=
```

Comment:
It's possible we might prefer the hypothesis instead to have type: `h : (f : L₁ → L₂) = g`.

I considered this but decided to mimic the signature of [the corresponding result](https://github.com/leanprover-community/mathlib/blob/5d35e626086c4e1639e49cd7cf574bc48592f554/src/algebra/module/basic.lean#L284) for linear maps.

More than happy to change things if others feel differently.